### ISSUE TITLE
Is tabletop still dying?

### ISSUE THREAD

**Issue by riquim** [jsoma/tabletop#192]
I read that tabletop is being killed by Google, however I can see it is still working.

It is the solution I need and I think I would be able to use it with my extremely low knowledge thanks to the good documentation there is around, so I hope it is not really dying.

I don't completely understand [issue 189 solution](https://github.com/jsoma/tabletop/issues/189) and I prefer not to use Papa Parse because I don't want to load JQuery everytime someone loads my website.

[This](https://css-tricks.com/creating-an-editable-webpage-with-google-spreadsheets-and-tabletop-js/) example still works like a charm and I will follow it, hoping it still will work for several months!

thanks for the great work!

**Comment by d3netxer** [comment]
it is still working for me as of now as well, so is there a countdown or something?

**Comment by MappingSteve** [comment]
@riquim
> I prefer not to use Papa Parse because I don't want to load JQuery

Papa Parse *does not require jQuery* - there are *no dependencies* (quoting [their website](https://www.papaparse.com/)). It *can* be used via jQuery, but it can also be used *by itself*. [I have not tried it yet; just quoting what I read.]

> issue 189

Perhaps try it *without* an API key - the earlier, simpler, instructions for how to use - giorgio's July 13 comment on that thread says it started working for him again.

**Comment by cheelay** [comment]
According to this link, they extended the migration deadline to January 26, 2021. I'm very glad!

[https://cloud.google.com/blog/products/g-suite/migrate-your-apps-use-latest-sheets-api ](url)

**Comment by mike-seekwell** [comment]
https://cloud.google.com/blog/products/g-suite/migrate-your-apps-use-latest-sheets-api

**Comment by cheelay** [comment]
They extended the deadline again: 

"UPDATE: To allow developers additional time to migrate their applications, we've extended the deadline for the turndown of the v3 API until June 8, 2021."

**Comment by cheelay** [comment]
Did anyone experience breakage after June 8th? My site is still working.

**Comment by datramt** [comment]
I too would like to know if this project is still alive. It would be incredibly useful for an upcoming project if so. It seems like most of the examples at the bottom of the readme are no longer online.

**Comment by cheelay** [comment]
They keep pushing back the date for turning off the v3 API support. Now their website says August 2nd is the final date.
https://cloud.google.com/blog/products/g-suite/migrate-your-apps-use-latest-sheets-api

**Comment by cheelay** [comment]
Seems like the killing is complete. My tabletop projects have stopped functioning.

**Comment by datramt** [comment]
Thanks @cheelay. Also, bravo to @jsoma for tabletop while it lasted!

**Comment by mroswell** [comment]
> I prefer not to use Papa Parse because I don't want to load JQuery everytime someone loads my website.

Papa Parse has no dependencies.

**Comment by levihernan** [comment]
Is there any fix to it or tabletop just died? :(

**Comment by davidcastillo** [comment]
yes, it seems it broken now.. when running the URL I get this message from google sheets

"Google logo
Sorry, unable to open the file at this time.

Please check the address and try again.

Get stuff done with Google Drive

Apps in Google Drive make it easy to create, store and share online documents, spreadsheets, presentations and more.

Learn more at drive.google.com/start/apps."

**Comment by d3netxer** [comment]
and its dead, no more public json endpoint of Google sheet

**Comment by fm9610** [comment]
Can the code be changed to conform with Google sheet API v4?

**Comment by d3netxer** [comment]
I don't know, I tried reading the Google sheet API v4 documentation but I didn't see any examples on how to make a JSON endpoint from a Google Sheet.

**Comment by levihernan** [comment]
`https://sheets.googleapis.com/v4/spreadsheets/SHEET_ID/values/SHEET_TAB_NAME/?key=API_KEY;`

you can get your `API_KEY` in https://console.cloud.google.com/apis/credentials

there's not much need for tabletop anymore

**Comment by levihernan** [comment]
if anyone is available to perform a PR changing tabletop completely for this API endpoint, it would save thousands of projects worldwide `<3`

**Comment by fm9610** [comment]
Thanks levihernan for the suggestion, but is there any way to parse the json result string back to the showInfo(data) used in tabletop?
